src/expire.c

```diff
@@ -584,14 +588,31 @@ void ttlGenericCommand(client *c, int output_ms) {
     }
 }
 
-/* TTL key */
+/* TTL key [ABS] */
 void ttlCommand(client *c) {
-    ttlGenericCommand(c, 0);
+    ttlGenericCommand(c, 0, 0);
 }
 
-/* PTTL key */
+/* PTTL key [ABS] */
```

Comment:
What's this cool flag do?

Reply:
lol this cool flag changed the meaning of the command so is cold-bloodedly discarded now.

Removed.